cdk/src/functions/living-docs-dashboard-generator.ts

```diff
@@ -1,8 +1,9 @@
-import AWS from 'aws-sdk';
+import { CloudWatch } from '@aws-sdk/client-cloudwatch';
+import { SSM } from '@aws-sdk/client-ssm';
 import endent from 'endent';
 
-const ssm = new AWS.SSM();
-const cloudwatch = new AWS.CloudWatch();
+const ssm = new SSM();
```

Comment:
You are still using the v2 compatible style. Maybe it makes sense to switch to command/send style to be consistent across superwerker.